cmd/stack_create_test.go

```diff
@@ -214,3 +214,36 @@ func TestStackAlreadyExists(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestStackCreateMissingArguments(t *testing.T) {
+	args := []string{"stack", "create"}
+	_, err := cmdtest.RunAppsodyCmd(args, ".", t)
+
+	if err == nil {
```

Comment:
Rather than blindly testing that an error is being returned, it'd be wiser to confirm that the error being returned is what we are expecting, because it could have returned for a completely unrelated issue and go unnoticed. Instead I would suggest something like:
```
if err != nil {
	if !strings.Contains(err.Error(), "<your expected error here>") {
		t.Errorf("String \"<your expected error here>\" not found in output: '%v'", err.Error())
	}
} else {
	t.Error("Expected an error to be returned from command, but error was nil")
}
```